### ISSUE TITLE
Not Recognizing Spotify as Installed

### ISSUE THREAD

**Issue by sevnee** [Xeroday/Spotify-Ad-Blocker#173]
Every time i start up EZBlocker it tells me i do not have Spotify installed, or have the window's store version. It still tells me this even after I reinstall Spotify through the button EZBlocker gives me to do so.

**Comment by andreluigo** [comment]
The same is happening here. Did you find a solution?

**Comment by sevnee** [comment]
> The same is happening here. Did you find a solution?

Nope! I've just been listening to Spotify from the web player with a browser ad-blocker in the meantime. But I haven't been able to get EZBlocker to work at all.

**Comment by Olit2000** [comment]
I got a similar problem: I want to enable banner-ads-blocking but this needs admin rights, so i quit EZBlocker and restart it with admin rights, but then it says Spotify isnt installed (which is true for the admin user but this user does not need Spotify)
I suggest a "ignore" Button as a workaround or for my special case an elevation promt inside EZBlocker so it has detected that Spotify is installed and then update the hosts file.
